docs/developer_setup.md

```diff
@@ -6,10 +6,10 @@ WSL2 should probably work, but has not been extensively tested.
 ## Pipeline software
 
 1. Install [Nix](https://nixos.org/) using the [Determinate Installer](https://github.com/DeterminateSystems/nix-installer#usage).
-2. Install [direnv](https://direnv.net/docs/installation.html).
-3. `git clone https://github.com/STFC-ICD-Research-and-Design/supermusr-data-pipeline`
-4. `cd supermusr-data-pipeline`
-5. `direnv allow`
+1. Install [direnv](https://direnv.net/docs/installation.html).
+1. `git clone https://github.com/STFC-ICD-Research-and-Design/supermusr-data-pipeline`
+1. `cd supermusr-data-pipeline`
+1. `direnv allow`
```

Comment:
Was the issue here the fact that some documents were auto numbered and some not?

If so I would prefer explicit numbering throughout.

Reply:
The default ordered list style in [this rule](https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md029---ordered-list-item-prefix) is 'one'. I have reverted to the original numbering style and have updated the style to 'ordered' inside `.mdl_style.rb`.